### PR TITLE
Updated KtCheckbox to support color strings

### PR DESCRIPTION
**Pull request by Goldziher**

### CODE REVIEW

File: packages/kotti-checkbox/src/Checkbox.vue [hunk 1]
```diff
@@ -17,11 +17,18 @@ export default {
...
 	computed: {
-	        hasLabel() {
-			return !(this.label===null || this.label === undefined)
+		hasLabel() {
+			return !(this.label === null || this.label === undefined)
+		},
+		iconColor() {
+			return {
+				backgroundColor: this.color ? this.color : 'none',
+				borderColor: this.color ? this.color : 'none',
```
Comment (amrdraz) [inline packages/kotti-checkbox/src/Checkbox.vue]: should none be the default?

Comment (amrdraz) [reply packages/kotti-checkbox/src/Checkbox.vue]: also you need to add an example in the docs

Comment (Goldziher) [reply packages/kotti-checkbox/src/Checkbox.vue]: example added. None should be the default.

Comment (FlorianWendelborn) [reply packages/kotti-checkbox/src/Checkbox.vue]: Colors should consume the KtTheme by default. Anything else means a shitload of work for any app trying to use consistent colors.

Comment (ostoc) [reply packages/kotti-checkbox/src/Checkbox.vue]: Actually there is also a requirement to change the color of the select box. if you knew the design in `AdminPanel` there is colorful checkbox to enable different graph.

**Pull request closed without merging**